mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -150,6 +181,32 @@ where
     }
 }
 
+pub async fn retry_subscriptions(
+    retries: UnboundedReceiver<SubscribeTo>,
```

Comment:
Why are we choosing unbounded?

Reply:
no specific reason. we could use either of the channels in this case

Reply:
Are we expecting a lot of simultaneous retries? As unbounded will take up more memory:

```
Note that the amount of available system memory is an implicit bound to the channel. Using an unbounded channel has the ability of causing the process to run out of memory. In this case, the process will be aborted.
```